libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts

```diff
@@ -108,27 +107,6 @@ describe("Date filtering on insight", () => {
         },
     );
 
-    it("disable date filter", { tags: ["checklist_integrated_tiger"] }, () => {
-        widget.waitChartLoaded();
-        widgetConfiguration
-            .open()
-            .openConfiguration()
-            .isDateDatasetDropdownExist(true)
-            .disableDateFilter()
-            .isDateDatasetDropdownExist(false);
-    });
-
-    it("enable date filter", { tags: ["checklist_integrated_tiger"] }, () => {
-        widget.waitChartLoaded();
-        widgetConfiguration
-            .open()
-            .openConfiguration()
-            .disableDateFilter()
-            .isDateDatasetDropdownExist(false)
-            .enableDateFilter()
-            .isDateDatasetDropdownExist(true);
-    });
-
```

Comment:
Remove these tests because it's already covered in Describe "Date filter with editmode"